backend/Dockerfile

```
@@ -3,7 +3,7 @@ FROM python:3.7-slim
 ADD tools /src/tools
 ADD backend /src/backend
 
-RUN cd /src/tools && python setup.py install
-RUN cd /src/backend && python setup.py install
+RUN cd /src/tools && pip install --disable-pip-version-check --no-cache-dir --quiet .
+RUN cd /src/backend && pip install --disable-pip-version-check --no-cache-dir --quiet . 
```

Please remove the trailing whitespace

```suggestion
RUN cd /src/backend && pip install --disable-pip-version-check --no-cache-dir --quiet .
```